labs-ios-starter/Model/Map.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Map.swift
+//  labs-ios-starter
+//
+//  Created by Jarren Campos on 2/10/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import MapKit
+
```

Comment:
Same here, documentation

Reply:
Added documentation for clarification. Thanks for the suggestion!